### ISSUE TITLE
Testimonals: Keyboard focus should have a noticeable outline

### ISSUE THREAD

**Issue by jeromehardaway** [Vets-Who-Code/vets-who-code-app#360]
Requirement: Interactive elements that can be navigated using the keyboard should be surrounded by a visual outline whenever they are focused.

Make each component tabbable
